modules/administration/pages/openscap.adoc

```diff
@@ -10,7 +10,11 @@ It allows you to schedule and view compliance scans for any client.
 OpenSCAP auditing is not available on Salt SSH clients.
 ====
 
-To use openSCAP, you will need the [systemitem]``spacewalk-oscap`` package installed on the clients you want to audit.
+[IMPORTANT]
+====
+Scanning clients can consume a lot of memory on both the server and the client being scanned.
```

Comment:
on the server too ? i don't think i said that ?

where did you get this info ?